### PR TITLE
[dev] [No PBI] Select onInputChange Prop

### PR DESCRIPTION
**Pull request by razmik-medoxi**

### CODE REVIEW

**Comment by groberts314** [comment]
@razmik-medoxi What functionality does this enable?

File: src/inputs/select/select.jsx [hunk 1]
```diff
@@ -192,6 +196,7 @@ const defaultProps = {
     noResultsText: 'No results found',
     onChange: null,
     onClose: undefined,
+    onInputChange: null,
```
Comment (groberts314) [inline src/inputs/select/select.jsx:199]: NAB: `undefined` is probably better here than `null`, not that it matters much.
I guess I like `null` as a default for _data_ (like this prop can be some specific object/type, or `null`), but for things like optional CSS classes or styles or event handler functions, I think `undefined` is appropriate.

**Comment by razmik-medoxi** [comment]
> @razmik-medoxi What functionality does this enable?

@groberts314 I'm exposing react-select's `onInputChange` prop, I need to get select's input value to trigger an API request and get the events list

**Comment by groberts314** [comment]
> I'm exposing react-select's onInputChange prop, I need to get select's input value to trigger an API request and get the events list

How is that different than the ordinary `onChange` event?

**Comment by razmik-medoxi** [comment]
> > I'm exposing react-select's onInputChange prop, I need to get select's input value to trigger an API request and get the events list
> 
> How is that different than the ordinary `onChange` event?

`onChange` is for selectedOption changes, this one is for the input changes

**Comment by groberts314** [comment]
> `onChange` is for selectedOption changes, this one is for the input changes

So, this event fires if someone types in the input versus picking something from the dropdown menu?

**Comment by razmik-medoxi** [comment]
> > `onChange` is for selectedOption changes, this one is for the input changes
> 
> So, this event fires if someone types in the input versus picking something from the dropdown menu?

exactly

**Comment by groberts314** [comment]
@razmik-medoxi These changes are available in [`@saddlebackchurch/react-cm-ui`](https://www.npmjs.com/package/@saddlebackchurch/react-cm-ui) version **`10.9.7`**.

**Pull request merged**